packages/@ionic/cli/src/commands/deploy/capacitor.ts

```diff
@@ -0,0 +1,66 @@
+import { BaseConfig } from "@ionic/cli-framework";
```

Comment:
To stay ahead of Capacitor 3 support coming soon, I copied some interface and class changes that will be coming later.

Reply:
ok